h/views/api/annotations.py

```diff
@@ -46,7 +46,7 @@ def search(request):
     schema = SearchParamsSchema()
     params = validate_query_params(schema, request.params)
 
-    _record_search_api_usage_metrics(params)
+    metrics.record_search_api_usage_metrics(params)
```

Comment:
I saw a PR go by after I left that had two different functions for the search params. We only need one and it should be called inside the Search class as that’s the common place where both activity pages and the api call into search. That being said, I would expect that the newrelic params call be removed entirely and moved to the Search class run method.